src/frontend/App/hooks/useMellomlagringFrittståendeSanitybrev.ts

```diff
@@ -0,0 +1,71 @@
+import { useApp } from '../context/AppContext';
+import { byggTomRessurs, Ressurs, RessursFeilet, RessursSuksess } from '../typer/ressurs';
+import { Dispatch, SetStateAction, useCallback, useEffect, useState } from 'react';
+import {
+    FlettefeltMedVerdi,
+    Fritekstområder,
+    MellomlagerRespons,
+    ValgteDelmaler,
+    ValgtFelt,
+} from '../../Komponenter/Behandling/Brev/BrevTyper';
+import { IMellomlagretBrev, MellomlagreSanitybrev } from './useMellomlagringBrev';
+
+export const useMellomlagringFrittståendeSanitybrev = (
+    fagsakId: string
+): {
+    mellomlagreSanitybrev: MellomlagreSanitybrev;
+    mellomlagretBrev: Ressurs<MellomlagerRespons | undefined>;
+    settMellomlagretBrev: Dispatch<SetStateAction<Ressurs<MellomlagerRespons | undefined>>>;
+} => {
+    const { axiosRequest } = useApp();
+    const [mellomlagretBrevRessurs, settMellomlagretBrevRessurs] = useState<
+        Ressurs<MellomlagerRespons | undefined>
+    >(byggTomRessurs());
+
+    const sanityVersjon = '1';
+
+    const mellomlagreSanitybrev = (
+        flettefelt: FlettefeltMedVerdi[],
+        valgteFelt: ValgtFelt,
+        valgteDelmaler: ValgteDelmaler,
+        fritekstområder: Fritekstområder,
+        brevmal: string
+    ): void => {
+        axiosRequest<string, IMellomlagretBrev>({
+            method: 'POST',
+            url: `/familie-ef-sak/api/brev/mellomlager/fagsak/${fagsakId}`,
+            data: {
+                brevverdier: JSON.stringify({
+                    flettefeltFraMellomlager: flettefelt,
+                    valgteFeltFraMellomlager: valgteFelt,
+                    valgteDelmalerFraMellomlager: valgteDelmaler,
+                    fritekstområderFraMellomlager: fritekstområder,
+                }),
+                brevmal,
+                versjon: sanityVersjon,
+            },
+        });
+    };
+
+    const hentMellomlagretBrev = useCallback(
+        () =>
+            axiosRequest<MellomlagerRespons | undefined, null>({
+                method: 'GET',
+                url: `/familie-ef-sak/api/brev/mellomlager/fagsak/${fagsakId}`,
+            }).then((res: RessursSuksess<MellomlagerRespons | undefined> | RessursFeilet) => {
+                settMellomlagretBrevRessurs(res);
+            }),
+        // eslint-disable-next-line
+        [fagsakId]
+    );
+
+    useEffect(() => {
+        hentMellomlagretBrev();
+    }, [fagsakId, hentMellomlagretBrev]);
```

Comment:
Hvorfor må denne lytte på `hentMellomlagretBrev`?

Reply:
I praksis ingen grunn til den må med 🤷 Men eslint klager på missing dependency hvis den ikke er med, så tenker det er ryddigere å inkludere den enn å disable eslint for den linja. Den trigger ingen unødvendige kall.